akka-contrib/src/main/scala/akka/contrib/circuitbreaker/askExtensions.scala

```diff
@@ -0,0 +1,94 @@
+package akka.contrib.circuitbreaker
+
+import akka.actor.{ ActorSelection, Actor, ActorRef }
+import akka.contrib.circuitbreaker.CircuitBreakerProxy.CircuitOpenFailure
+import akka.util.Timeout
+import scala.language.implicitConversions
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+final class OpenCircuitException extends Exception("Unable to complete operation since the Circuit Breaker Actor Proxy is in Open State")
```

Comment:
`RuntimeException` - because otherwise it's a checked one for JavaDSLs, not as nice to work with